lib/runtime-c-api/tests/test-import-function-callinfo.c

```diff
@@ -0,0 +1,132 @@
+#include <stdio.h>
+#include "../wasmer.h"
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+
+static bool print_str_called = false;
+static int memory_len = 0;
+static int ptr_len = 0;
+static char actual_str[14] = {};
+static int actual_context_data_value = 0;
+
+typedef struct {
+  int value;
+} context_data;
+
+struct print_str_context {
+    int call_count;
+};
+
+void print_str(struct print_str_context *local_context, uint64_t *args)
+{
+    local_context->call_count++;
+
+    wasmer_instance_context_t *ctx = (void *) args[0];
+    int32_t ptr = args[1];
+    int32_t len = args[2];
+    
+    const wasmer_memory_t *memory = wasmer_instance_context_memory(ctx, 0);
+    uint32_t mem_len = wasmer_memory_length(memory);
+    uint8_t *mem_bytes = wasmer_memory_data(memory);
+    for (int32_t idx = 0; idx < len; idx++)
+    {
+        actual_str[idx] = mem_bytes[ptr + idx];
+    }
+    actual_str[13] = '\0';
+    printf("In print_str, memory len: %d, ptr_len: %d\n, str %s", mem_len, len, actual_str);
+    print_str_called = true;
+    memory_len = mem_len;
+    ptr_len = len;
+
+    actual_context_data_value = ((context_data *) wasmer_instance_context_data_get(ctx))->value;
+}
+
+int main()
+{
+    wasmer_value_tag params_sig[] = {WASM_I32, WASM_I32};
+    wasmer_value_tag returns_sig[] = {};
+    struct print_str_context local_context = {
+        .call_count = 0
+    };
+
+    printf("Creating trampoline buffer\n");
+    wasmer_trampoline_buffer_builder_t *tbb = wasmer_trampoline_buffer_builder_new();
+    unsigned long print_str_idx = wasmer_trampoline_buffer_builder_add_callinfo_trampoline(
+        tbb,
+        (wasmer_trampoline_callable_t *) print_str,
+        (void *) &local_context,
+        3
```

Comment:
3 for the instance context + `print_str` arguments, is that correct?

I wonder if we should keep the instance context 🤔… it's for another PR anyway.